cli/scancommands.go

```diff
@@ -472,7 +475,7 @@ func CreateAuditCmd(c *components.Context) (string, string, *coreConfig.ServerDe
 
 	auditCmd.SetTargetRepoPath(addTrailingSlashToRepoPathIfNeeded(c)).
 		SetProject(getProject(c)).
-		SetIncludeVulnerabilities(c.GetBoolFlagValue(flags.Vuln) || shouldIncludeVulnerabilities(c)).
+		SetIncludeVulnerabilities(c.GetBoolFlagValue(flags.Vuln)).
```

Comment:
why did we take off the || shouldIncludeVulnerabilities(c) part only from Audit but not from Scan or DockerScan?

Reply:
because only in audit we are passing `git repo url` but you are right, we can remove those as well